src/components/testCaseLanding/qdm/TestCaseList.tsx

```diff
@@ -497,25 +498,35 @@ const TestCaseList = (props: TestCaseListProps) => {
         calculationOutput,
         callstackMap
       );
-    const excelData: Blob = await excelExportService.current.generateExcel(
-      testCaseDtos
-    );
-    var exportBlob = new Blob([excelData], {
-      type: "application/vnd.ms-excel",
-    });
-    const url = window.URL.createObjectURL(exportBlob);
-    const link = document.createElement("a");
-    link.href = url;
-    link.setAttribute(
-      "download",
-      `${measure.ecqmTitle}-v${measure.version}-QDM-TestCases.xls`
-    );
-    document.body.appendChild(link);
-    link.click();
-    setToastOpen(true);
-    setToastType("success");
-    setToastMessage("Excel exported successfully");
-    document.body.removeChild(link);
+
+    excelExportService.current
```

Comment:
Looks good, maybe add test coverage later?

Reply:
Yup We should work on that later, I just started looking into this to un-block 6729